### PR TITLE
Upgrade to Polymer 3

### LINKED ISSUE #33: Upgrade to Polymer 3

### PR DESCRIPTION
**Pull request by cristinecula**
Fixes #33

### CODE REVIEW

**Comment by cristinecula** [comment]
@nomego @megheaiulian

File: demo/form-dialog.html [hunk 1]
```diff
@@ -6,30 +6,36 @@
...
+	</script>
 </head>
 <body unresolved>
-
 	<h3>cosmoz-dialog in root stacking context</h3>
 	<demo-snippet>
 		<template>
-			<paper-button raised onclick="dialogInRootStackingContext.open()">open dialog</paper-button>
-			<cosmoz-form-dialog with-backdrop id="dialogInRootStackingContext">
+			<paper-button raised="" onclick="dialogInRootStackingContext.open()">open dialog</paper-button>
```
Comment (nomego) [inline demo/form-dialog.html]: `=""` should not be needed for bool attributes?

**Review by nomego — APPROVED** [review]

File: demo/form-dialog.html (hunk 1 above)
Comment (cristinecula) [reply demo/form-dialog.html]: It's not needed; it's added erroneously by polymer-modulizer.

**Comment by cristinecula** [comment]
@nomego ready to release!

**Pull request merged**